build/Targets/Dependencies.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--Generated file, do not directly edit.  Run "RepoUtil change" to regenerate-->
   <PropertyGroup>
-    <ManagedEsentVersion>1.9.2.0</ManagedEsentVersion>
+    <ManagedEsentVersion>1.9.2</ManagedEsentVersion>
```

Comment:
What version should we update to? 

I made this change because the actual NuGet package version is 1.9.2.  It seemed to be a typo to have it be 1.9.2.0 and all of the file based references were broken bbecause the directory layout was different.